src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -89,19 +86,116 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
         addHighlightSet: true,
       });
       closeStepperModal();
+      const handlePublishTrackEvent = () => {
+        const trackInfo = {
+          stepper_modal: {
+            current_step: currentStep,
+            current_step_position: steps.indexOf(currentStep) + 1,
+          },
```

Comment:
We can already pretty much assume which step the user is on at the time of this event firing (i.e., they'd have to be on the final confirm step) so I think the `stepper_modal` data may be unnecessary.